openreview/conference/templates/bidWebfield.js

```diff
@@ -10,7 +10,35 @@ var BID_ID = '';
 var SUBJECT_AREAS = '';
 var AFFINITY_SCORE_ID = '';
 var CONFLICT_SCORE_ID = '';
+
+// Bid status data
+var activeTab = 0
 var noteCount = 0;
+var conflictIds = [];
+var bidsByNote = {};
+var bidsById = {
+  'Very High': [],
+  'High': [],
+  'Neutral': [],
+  'Very Low': [],
+  'Low': []
+};
+var sections = [];
+var scoresByName = {
+  'TPMS': 'thecvf.com/ECCV/2020/Conference/Reviewers/-/TPMS_Score',
+  'Affinity': 'thecvf.com/ECCV/2020/Conference/Reviewers/-/Affinity_Score'
+}
```

Comment:
I think this is fine for now, but I think we should come up with a more general method before this is merged into master.